js/src/ui/SelectionList/selectionList.css

```diff
@@ -43,13 +46,17 @@
 }
 
 .selected {
-  box-shadow: inset 0 0 0 2px rgb(255, 255, 255);
+  border-color: rgba(255, 255, 255, 0.25);
   filter: none;
+
+  &.default {
+    border-color: rgba(255, 255, 255, 0.75);
+  }
 }
 
 .unselected {
-  filter: grayscale(100%);
-  opacity: 0.5;
+  filter: grayscale(10%);
```

Comment:
Me too, actually. Had to contain it a bit to not look totally disabled, yet, but more subtle.